faerie_david/src/Main.cpp

```diff
@@ -9,42 +9,18 @@
 
 // Standard Includes
 #include <Arduino.h>
-#include <Servo.h>  // For SCABBARD servo (unused)
 
 #include <cmath>  // for abs()
-#include <iostream>
-#include <string>
 #include <vector>
 
 // Our own resources
-#include "AstraCAN.h"
-#include "AstraMotors.h"
-// #include "AstraSensors.h" // Unused, only sensor on FAERIE is SHT
-#include "Adafruit_SHT31.h"
-#include "TeensyThreads.h"
+#include "FAERIE.h"
```

Comment:
Random thought I've got. (Cannot access Discord rn so I'm going to put it here)

I think, pending any other changes we make to the way we do it, I think we should put all the pin file (FAERIE.h, CORE.h, etc...) into a sub-folder in the include/ folder in rover-Embedded-Lib if possible. Think it'd be clean and separate them from the other headers and files we have in the root include folder.

Probably will find use in more subfolders as we grow the library.

Reply:
The only annoying thing is that you'll have to specify any subfolders in the include statements (`#include "projects/FAERIE.h"`) but I'm not opposed to that